.travis.yml

```diff
@@ -13,8 +13,9 @@ before_script:
   - export DJANGO_SECRET_KEY=foobarbaz
 script:
   - cd vms
-  - python manage.py syncdb --noinput
-  - python manage.py migrate --noinput --traceback --settings=vms.settings
+  - flake8 . 
+  - python3.6 -m manage.py syncdb --noinput
+  - python3.6 -m manage.py migrate --noinput --traceback --settings=vms.settings
```

Comment:
👋 Is there any specific advantage of not using `makemigrations` and `migrate` but using -n and using `syncdb` instead?

Reply:
it shoudl be `makemigrations` and `migrate` please. `syncdb` is deprecated.

Reply:
actually thanks for leaving this note real quick @tonythomas01 ! and keeping an 👀

Reply:
even though, this `python3.6` can be replaced with just `python` right ?

Reply:
@tonythomas01 should I change it to makemigrations and migrate?

Reply:
yes, please.

Reply:
@tonythomas01 Done! Please review.